content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -50,15 +51,36 @@ Install Istio and expose the control plane so that your virtual machine can acce
 
 1. Install Istio.
 
+    {{< tabset category-name="istio-install" >}}
+
+    {{< tab name="Default" category-value="default-install" >}}
+
     {{< text bash >}}
     $ istioctl install
     {{< /text >}}
 
-    {{< tip >}}
-    To enable experimental [VM auto-registration](/docs/examples/virtual-machines/autoregistration): `istioctl install --set values.global.pilot.env.PILOT_ENABLE_WORKLOAD_ENTRY_AUTOREGISTRATION=true`.
-    {{< /tip >}}
+    {{< /tab >}}
 
-1. Expose the control plane using the provided sample configuration.
+    {{< tab name="Automated WorkloadEntry Creation" category-value="autoreg-install" >}}
+
+    {{< warning >}}
+    This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+    considered `pre-alpha`.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ istioctl install --set values.global.pilot.env.PILOT_ENABLE_WORKLOAD_ENTRY_AUTOREGISTRATION=true
+    {{< /text >}}
+
+    {{< /tab >}}
+
+    {{< /tabset >}}
+
+1. Deploy the east-west gateway and expose the control plane using the provided sample configuration.
+
+    {{< text bash >}}
+    $ @samples/multicluster/gen-eastwest-gateway.sh@ --single-cluster | istioctl install -y -f -
+    {{< /text >}}
```

Comment:
Normally I see instructions and then a `text bash` showing the command(s) for the instructions. Here I see one instructions and two separate `text bash`. Can they be combined into one `text bash` for the single instruction or can we add additional text for the second text bash?

Reply:
Split this at the `and`